eventmesh-common/src/main/java/org/apache/eventmesh/common/config/CommonConfiguration.java

```diff
@@ -123,12 +123,12 @@ private String checkNotEmpty(String key) {
         return value;
     }
 
-    private String checkNumeric(String key) {
-        String value = configurationWrapper.getProp(key);
+    private String checkNumeric() {
+        String value = configurationWrapper.getProp(ConfKeys.KEYS_EVENTMESH_SYSID);
```

Comment:
I think the method `private String checkNumeric(String key)`  here is introduced as a common method to check the numeric property,  if we remove the key parameter and just use `checkNumeric()` to get specific property,  the method name would be very confusing.

Reply:
Makes sense